contracts/vesting.token/include/vesting.token/vesting.token.hpp

```diff
@@ -0,0 +1,90 @@
+// vesting.token.hpp
+
+#pragma once
+
+#include <eosio/action.hpp>
+#include <eosio/eosio.hpp>
+#include <eosio/asset.hpp>
+#include <eosio/system.hpp>
+#include <eosio/singleton.hpp>
+
+namespace vestingtoken {
+    using eosio::action_wrapper;
+    using eosio::asset;
+    using eosio::name;
+    using std::string;
+    using eosio::check;
+
+    typedef eosio::singleton<"salesdate"_n, eosio::time_point_sec> sales_start_date;
+    typedef eosio::singleton<"launchdate"_n, eosio::time_point_sec> launch_date;
```

Comment:
both these values should be in one singleton. that reduces the number of database lookups, improving efficiency.